packages/runtime/lib/runtime.js

```diff
@@ -111,6 +111,14 @@ class Runtime extends EventEmitter {
     this.#servicesIds = config.services.map(service => service.id)
     this.#workers.configure(config.services, this.#configManager.current.workers, this.#isProduction)
 
+    if (this.#isProduction) {
+      this.#env['PLT_DEV'] = false
```

Comment:
These must be strings (`'false'`), not booleans

Reply:
Fixed! Anyway they arrive a string in the worker thread